docs/src/main/paradox/use-cases.md

```diff
@@ -3,3 +3,29 @@
 Akka Projections is intended for the following primary use cases. It is not limited to these use cases,
 because it is designed to be flexible in the way different sources and targets of the projections can be
 composed.  
+
+## Command Query Responsibility Segregation (CQRS)
+
+CQRS is a pattern that is used for separating operations that write data from those that read the data.
+Even though it's not necessary to use event sourcing for CQRS that is a common approach. Event sourced
+entities can only be queried by their identifier and therefore it's often necessary to also have a
+different model that can be queried. From the events you can maintain one or more representations that are
+optimized for queries, for example in a relational database.
+
+![use-case-cqrs.png](./images/use-case-cqrs.png)
+
+## Service to service communication
+
+When designing your microservices, each service should own its data and have direct access to the database.
+Other services should then use the Service API to interact with the data. There must be no sharing of databases
+across different services since that would result in a too-tight coupling between the services. In this way,
+each microservice operates within a clear boundary, similar to the [Bounded Context](https://martinfowler.com/bliki/BoundedContext.html)
+strategic pattern in Domain-driven Design.
+
+One service is publishing events:
+
+![use-case-service1.png](./images/use-case-service1.png)
```

Comment:
The diagrams are a great addition.
Some possible improvements:
- Use thicker lines for all the black and white ones
- If in Keynote use the DB picture rather than just the shape
- For Kafka use their logo? 
E.g.

![image](https://user-images.githubusercontent.com/1866779/82017791-d5c7b400-967b-11ea-89ed-2c9d5a800683.png)

![image](https://user-images.githubusercontent.com/1866779/82017811-df511c00-967b-11ea-8f8b-d4998dbd20b3.png)

Reply:
thanks, updated images again

Reply:
Looks good